packages/lib/services/interop/InteropService_Exporter_Md.test.js

```diff
@@ -67,7 +68,53 @@ describe('services_InteropService_Exporter_Md', function() {
 		expect(Object.keys(exporter.context().notePaths).length).toBe(3, 'There should be 3 note paths in the context.');
 		expect(exporter.context().notePaths[note1.id]).toBe('folder1/note1.md');
 		expect(exporter.context().notePaths[note2.id]).toBe('folder1/note2.md');
-		expect(exporter.context().notePaths[note3.id]).toBe('folder2/note3.md');
+		expect(exporter.context().notePaths[note3.id]).toBe('folder2/note3.html');
```

Comment:
We do check for specific things (like properly encoded file paths), but adding a title check is a good idea. I'm holding off on doing it for now because, as I mentioned in the other comment, I want to remove titles from the body of the exported note.